Header header

#Setpoints for the thrusters 0 off, 255 max
uint8 thruster1
uint8 thruster2
uint8 thruster3
uint8 thruster4
uint8 thruster5
uint8 thruster6
